docs/2.guide/2.directory-structure/1.components.md

```diff
@@ -166,6 +166,38 @@ export default defineNuxtConfig({
 })
 ```
 
+## NPM Packages
+
+If you want to auto-import components from an npm package, you can use [`addComponent`](/docs/api/kit/components#addcomponent) in a Nuxt hook to register them.
+
+::code-group
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  hooks: {
```

Comment:
I would recommend doing this as a local module - could we move this into `~/modules/register-component.ts`...?

Reply:
Thanks for the suggestion! 🙏 Updated 👍